test/tattr.c

```diff
@@ -4542,7 +4543,7 @@ test_attr_corder_create_compact(hid_t fcpl, hid_t fapl)
 {
     hid_t    fid;                     /* HDF5 File ID            */
     hid_t    dset1, dset2, dset3;     /* Dataset IDs            */
-    hid_t    my_dataset;              /* Current dataset ID        */
+    hid_t    my_dataset = 0;          /* Current dataset ID        */
```

Comment:
The invalid value for hid_t types is H5I_INVALID_HID

Reply:
Done.